### PR TITLE
Add worker rejected cache block request metrics

### PR DESCRIPTION
**Pull request by qian0817**
### What changes are proposed in this pull request?

Add cache rejected metrics.

### Why are the changes needed?

Add cache rejected metrics.

### Does this PR introduce any user facing changes?

Add metrics `Worker_CacheRejectedBlocks`.

### CODE REVIEW

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -2119,6 +2119,12 @@ public static String getSyncMetricName(long mountId) {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_CACHE_REJECTED_BLOCKS =
+      new Builder("Worker.CacheRejectedBlocks")
+          .setDescription("Total number of cache rejected blocks in this worker")
```
Comment (jiacheliu3) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:2124]:
```suggestion
          .setDescription("Total number of rejected cache block requests on the worker")
```

**Review by jiacheliu3 — APPROVED** [review]
LGTM with a small comment, thanks for the contribution!

**Review by maobaolong — APPROVED** [review]
This kind about metrics improvement PRs are always welcome to me. LGTM

How about update the title to `Add worker pagestore cache rejected metrics`

**Comment by qian0817** [comment]
> This kind about metrics improvement PRs are always welcome to me. LGTM
> 
> How about update the title to `Add worker pagestore cache rejected metrics`

done.

**Comment by maobaolong** [comment]
Jenkins-bot, merge this please!

**Comment by maobaolong** [comment]
alluxio-bot, merge this please

**Comment by maobaolong** [comment]
alluxio-bot, merge this please

**Comment by maobaolong** [comment]
alluxio-bot, merge this please

**Pull request merged**